systemd/mdmonitor.service

```diff
@@ -11,7 +11,21 @@ DefaultDependencies=no
 Documentation=man:mdadm(8)
 
 [Service]
-Environment=  MDADM_MONITOR_ARGS=--scan
-EnvironmentFile=-/run/sysconfig/mdadm
-ExecStartPre=-/usr/lib/mdadm/mdadm_env.sh
-ExecStart=BINDIR/mdadm --monitor $MDADM_MONITOR_ARGS
+# For Maintainers:
+# We need to ensure that the mdmonitor configuration aligns with the guidelines provided
+# in the man page for users.
+# /etc/sysconfig/mdadm, /etc/sysconfig/mdmonitor, or any other similar configuration file should
+# not be supported because they are not part of upstream (they are not described for users).
```

Comment:
Maybe "they're not user manageable"?

Reply:
This is not true, they are manageable but the implementation is less clear, (man page might be inconsistent with the service implementation). I will refer to man pages.